src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala

```diff
@@ -1598,6 +1597,102 @@ class TezosDatabaseOperationsTest
         Map("level" -> Some(1), "proto" -> Some(1))
       )
     }
+
+    "should aggregation with COUNT function" in {
+      val feesTmp = List(
+        FeesRow(0, 2, 4, new Timestamp(0), "kind"),
+        FeesRow(0, 4, 8, new Timestamp(1), "kind"),
+        FeesRow(0, 3, 4, new Timestamp(2), "kind")
+      )
+
+      val aggregate = Some(
+        Aggregation("medium", AggregationType.count, None)
+      )
+
+      val populateAndTest = for {
+        _ <- Tables.Fees ++= feesTmp
+        found <- sut.selectWithPredicates(Tables.Fees.baseTableRow.tableName, List("low", "medium", "high"), List.empty, List.empty, 3, aggregate)
```

Comment:
Using named arguments in the call can help out to figure out what these values represents, e.g.

```scala
sut.selectWithPredicates(
  table = ...,
  columns = ...,
  predicates = ...,
  ordering = ...
  ...
)
```

Reply:
changed